src/scm/bitbucket.js

```diff
@@ -196,111 +203,126 @@ class Bitbucket {
     const isPrivate = $('#new-repo-type').val() !== 'public';
     const payload = {
       scm: 'git',
-      description : desc,
+      description: desc,
       is_private: isPrivate
     }
     if (!name || name === '') return;
     return this.getAccessToken()
-    .then(() => {
-      return $.ajax({
-        url: `${this.baseUrl}/repositories/${owner}/${name}`,
-        headers: {
-          'Authorization': `Bearer ${this.accessToken}`
-        },
-        method: 'POST',
-        crossDomain: true,
-        dataType: 'json',
-        contentType: 'application/json',
-        data: JSON.stringify(payload)
-      })
-    })
-    .then(response => {
-      const repo = {
-        fullName : response.full_name
-      };
-      context.repo = repo;
-      Object.assign(context.bindRepo, { [context.id] : repo });
-      if (context.bindBranch[context.id]) {
-        delete context.bindBranch[context.id];
-      }
-      chrome.storage.sync.set({ bindRepo: context.bindRepo });
-      return response.full_name;
-    })
-    .then(repo => {
-      return this.commitFiles(repo, 'master', null, [{name: "README.md", content: "initialed by gas-github"}], null, 'initial commit')
       .then(() => {
-        return repo;
+        return $.ajax({
+          url: `${this.baseUrl}/repositories/${owner}/${name}`,
+          headers: {
+            'Authorization': `Bearer ${this.accessToken}`
+          },
+          method: 'POST',
+          crossDomain: true,
+          dataType: 'json',
+          contentType: 'application/json',
+          data: JSON.stringify(payload)
+        })
+      })
+      .then(response => {
+        const repo = {
+          fullName: response.full_name
+        };
+        context.repo = repo;
+        Object.assign(context.bindRepo, {
+          [context.id]: repo
+        });
+        if (context.bindBranch[context.id]) {
+          delete context.bindBranch[context.id];
+        }
+        chrome.storage.sync.set({
+          bindRepo: context.bindRepo
+        });
+        return response.full_name;
+      })
+      .then(repo => {
+        return this.commitFiles(repo, 'master', null, [{
+            name: "README.md",
+            content: "initialed by gas-github"
+          }], null, 'initial commit')
+          .then(() => {
+            return repo;
+          });
+      })
+      .catch((err) => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/15f4157afeba0560f3f12968d583a6f4fceda1cf/src/scm/bitbucket.js#L249"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>